osbuild/ostree.py

```diff
@@ -0,0 +1,90 @@
+import contextlib
+import json
+import os
+import tempfile
+
+from typing import List
+
+
+class Param:
+    def __init__(self, value_type, mandatory=False, default=None):
```

Comment:
How about:
`def __init__(self, value_type, *, mandatory=False, default=None):`

This forces call-sites to use named arguments, which makes them a lot nicer to read than just `true` or `false`.

Reply:
We discussed this separately. To make sure we have this documented: The unannotated star `*` does exactly what we expect: It forces the following parameters to be keyword-arguments. It has no other side-effects. It does not swallow other positional arguments.

We can adjust this later on as well. But I think forced keyword-arguments seems like a nice feature.

Reply:
I am on the fence here because I realized I wanted no a 'real' keyword argument but an normal optional argument with a default value. But I don't feel strongly about it here.